tarofreighter/chain_porter.go

```diff
@@ -1151,6 +1113,36 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage,
 		currentPkg.SendState = SendStateWaitTxConf
 		return &currentPkg, nil
 
+	// At this point, transaction broadcast is complete. We go on to wait
```

Comment:
The comment for the state before mentions "terminal" state, which I guess is no longer true now.